duckdns/README.md

```diff
@@ -79,7 +79,7 @@ IPv6 address manually.
 
 If you specify a URL here, contents of the resource it points to will be
 fetched and used as the address. This enables getting the address using
-a service like https://api6.ipify.org/ or https://ipv6.wtfismyip.com/text
+an external service.
```

Comment:
The problem with this change: It now doesn't tell the user anything. Giving examples helps to make clear what a user "could" use. "an external service" doesn't help a user.

Reply:
I think this is clearly an improvement over giving two example services that we know for sure **not** to work.